src/hotspot/share/nmt/memTracker.hpp

```diff
@@ -135,15 +135,15 @@ class MemTracker : AllStatic {
     }
   }
 
-  static inline void record_virtual_memory_release(address addr, size_t size) {
+  static inline void record_virtual_memory_release(void* addr, size_t size) {
     assert_post_init();
     if (!enabled()) return;
     if (addr != nullptr) {
       VirtualMemoryTracker::remove_released_region((address)addr, size);
```

Comment:
What about here?

Reply:
That is not the plan, no. I believe that the address typedef is there to signify that we are concerned with storing addresses, but not dereferencing them.